tv_apps/app-deck/style/app_deck.css

```diff
@@ -77,46 +80,46 @@ section {
 .app-banner {
   display: inline-block;
   background-color: #999;
-  width: 35.55vh;
-  height: 20.18vh;
-  margin: 2.03vh 0;
+  width: 38.4rem;
+  height: 21.8rem;
+  margin: 2.2rem 1.4rem;
   left: 0;
   position: absolute;
 }
 
 #app-deck-grid-view-container {
   width: 100%;
-  height: auto;
+  height: calc(100% - 48.8rem);
+  background-color: #e0e0e0;
 }
 
 #app-deck-grid-view {
-  margin: 5.55vh 6.04vw 0 6.04vw;
+  /* top = (6 - 5.8)rem, left = right = (14.2 - 2.6)rem */
+  margin: 0.2rem 11.6rem 0 11.6rem;
```

Comment:
The margin is not suggested in this case because it clips the first button while we focus it. Please use `padding` instead. That works better. Please try it.